app/graphql/types/query_type.rb

```diff
@@ -17,10 +17,16 @@ def get_availabilities(user_id:, status:)
         .order(start_date_time: :asc)
     end
 
+    field :get_user, Types::UserType, null: false, description: 'Returns a single user by id' do
+      argument :id, ID, required: true
+    end
+
     def get_users
       User.all
     end
 
+    field :get_users, [Types::UserType], null: false, description: 'Returns a list of users'
+
     def get_user(id:)
       User.find(id)
     end
```

Comment:
I reordered the fields to be next to their corresponding resolvers for organization

Reply:
Thanks! Just looks like get_user and get_users aren't by their own resolvers (switched around) @lkriffell